gradle/dependencies.gradle

```diff
@@ -15,17 +15,23 @@
  */
 
 def versions = [
-    kotlin    : "1.2.60",
-    support   : "27.1.0"
+    kotlin    : "1.3.10"
+]
+
+def androidx = [
+    annotations      : "androidx.annotation:annotation:1.0.0",
+    appcompat        : "androidx.appcompat:appcompat:1.0.0",
+    constraintLayout : "androidx.constraintlayout:constraintlayout:1.1.2",
+    v4               : "androidx.legacy:legacy-support-v4:1.0.0"
 ]
 
 def apt = [
     autoService : "com.google.auto.service:auto-service:1.0-rc4"
 ]
 
 def build = [
-    androidPlugin       : "com.android.tools.build:gradle:3.1.4",
-    buildToolsVersion   : "28.0.2",
+    androidPlugin       : "com.android.tools.build:gradle:3.2.1",
+    buildToolsVersion   : "28.0.3",
```

Comment:
nit: AGP should be able to manage this for you without needing to specify it

Reply:
neat